src/js/main.js

```diff
@@ -153,7 +161,25 @@ function fuzzySearch(){
         populateSearchSuggestions(searchString);
         var fuzz = new Fuse(suggestionList, options);
         var fuzzResult = fuzz.search(searchString);
-        populateSuggestionsBox(fuzzResult);
+        const highlightedResult = fuzzResult.map(({ item, matches }) => {
```

Comment:
sure you don't want to use fat arrow for callbacks? it is pretty standard now a days. There is no "this" context in these functions so it doesn't change anything, but the fat arrow gives very clear visual queues there.